business/workloads.go

```diff
@@ -108,6 +108,7 @@ func (in *WorkloadService) GetWorkloadList(criteria WorkloadCriteria) (models.Wo
 	}()
 
 	istioConfigCriteria := IstioConfigCriteria{
+		AllNamespaces:                 true,
```

Comment:
This change is showing a problem in the way the EnvoyFilters are filtered.

Probably this method is not 100% correct:
![image](https://user-images.githubusercontent.com/1662329/145402820-e27a93d7-cca0-4bdc-8be3-06dc52a8b3a4.png)

It was introduced before but now with this PR we can see the problem.

Perhaps for these object we may want to a) make the filter explictly (don't add to the result generic EF objects) or b) just ignore the EF for now, as those objects are typically low level, and rely the user to query them from the Istio Config only.